tests/test.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2022 Pablo Correa Gomez
```

Comment:
If you place copyright here, please also do so in the C test file and also mention the SPDX license

Reply:
Done!